command/sub8_launch/launch/subsystems/cameras.launch

```diff
@@ -1,4 +1,4 @@
 <launch>
-  <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/pt_grey_stereo.launch" />
-  <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/downward.launch" />
+  <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/front_cameras.launch" />
+  <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/down_cameras.launch" />
```

Comment:
Yes. Both looked good besides green bug and had correct frames in camera info and image topics